src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -59,7 +59,7 @@ public void testCons() {
 
     @Test
     public void testDupString() {
-        Assert.assertEquals("ababab", Utils.dupString("ab", 3));
+        //removed test because method is being removed
         Assert.assertEquals("aaa", Utils.dupString('a', 3));
```

Comment:
Ignore the previous comments, but separate this into 2 two tests, name one dupBytes and one dupChar and then rename that dupStrings to dupChar